tank_lab_to_nwb/convert_towers_task/convert_towers.py

```diff
@@ -51,9 +49,10 @@ def run_tower_conv(session, nwbfile_path):
 
             converter.run_conversion(nwbfile_path=nwbfile_path, metadata_dict=metadata, stub_test=True)
     else:
-        print(f"The folder ({session}) does not exist!")
+        print(f"The folder ({base_path}) does not exist!")
```

Comment:
Move checks like this to beginning of function, and they should throw an error. Then you can unindent the rest of the function